src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  static die(animal) {
+    Animal.alive = Animal.alive.filter(a => a !== animal);
```

Comment:
Bad practice to compare objects by references. You can filter by health value. Or assign to every animal a unique id and use it in the filter.

Reply:
Also, what is `a`?